src/DataFrame/Operations/Statistics.hs

```diff
@@ -249,7 +249,7 @@ imputeWith f col@(Col columnName) df = case interpret @b (filterJust columnName
         Left e -> throw e
         Right h ->
             if all (== h) (toList @b value)
-                then impute col h df
+                then DataFrame.Operations.Transformations.impute col h df
```

Comment:
Actually I'm stupid. Line 254 is supposed to be impute with. Not even supposed to be ambiguous.